bundle/src/main/java/com/adobe/acs/commons/fam/impl/ActionManagerImpl.java

```diff
@@ -144,7 +144,7 @@ public void withResolver(CheckedConsumer<ResourceResolver> action) throws Except
         resolver.setCurrentItem(currentPath.get());
         try {
             action.accept(resolver.getResolver());
-        } catch (Exception ex) {
+        } catch (Throwable ex) {
```

Comment:
sorry, I meant to comment on the *next* change, where the exception isn't thrown. Here I think you don't need the catch block at all and this should just be rewritten to 

   try {
       action.accept(resolver.getResolver());
   } finally {
       ... finally block...
   }